### PR TITLE
kubermatic-e2e pipeline integration

### LINKED ISSUE #1491: kubermatic-e2e pipeline integration
As we now have automated Conformance testing (with help of kubermatic-e2e) merged and tested and with acceptable time run (~15-20 minutes), we need to integrate it to pipline run it **after every master merge**.

TODO plan:
* build kubermatic/kubermatic-e2e image
* create bunch of secrets in our drone with cluster/node specs combinations, each for different cloud we'd like to e2e test
* modify drone.yml to run kubermatic-e2e on top of dev/cloud environment **after** each deploy

### PR DESCRIPTION
**Pull request by kron4eg**
**What this PR does / why we need it**:
Run kubernatic-e2e tests on every master merge

Fixes #1491

**Special notes for your reviewer**:
needs drone secrets: kubeconfig, cluster and node manifests

### CODE REVIEW

**Comment by kron4eg** [comment]
`.drone.jsonnet` was formatter with `jsonnet fmt`

File: .drone.jsonnet [hunk 1]
```diff
@@ -62,97 +83,111 @@ local drone = import 'drone/drone.libsonnet';
...
+        { source: 'kubeconfig_cloud', target: 'base64_kubeconfig' },
+        { source: 'aws_cluster_yaml', target: 'base64_cluster_yaml' },
+        { source: 'aws_node_yaml', target: 'base64_node_yaml' },
+      ],
+      commands: [
+        'mkdir /config',
+        'echo $BASE64_KUBECONFIG | base64 -d > /config/kubeconfig',
+        'echo $BASE64_CLUSTER_YAML | base64 -d > /manifests/cluster.yaml',
+        'echo $BASE64_NODE_YAML | base64 -d > /manifests/node.yaml',
+        'echo "let kubermatic redeploy" && sleep 30',
```
Comment (mrIncompetent) [inline .drone.jsonnet]: Doesent the `--wait` flag on helm already covers that?

File: .drone.yml [hunk 2]
```diff
@@ -37,8 +37,6 @@ pipeline:
     group: lint
     image: docker:dind
     pull: true
-    volumes:
-      - /var/run/docker.sock:/var/run/docker.sock
```
Comment (mrIncompetent) [inline .drone.yml]: Needs to be manually added back. The drone jsonnet lib does not support volumes yet, therefore we need to manually maintain it

Comment (kron4eg) [reply .drone.yml]: oh... didn't know about this and was suprized after first jsonnet run that produced file has diff. We should document that probably for future.

File: .drone.jsonnet (hunk 1 above)
Comment (kron4eg) [reply .drone.jsonnet]: removed that line

File: .drone.yml (hunk 2 above)
Comment (kron4eg) [reply .drone.yml]: Fixed that issue in jsonnet file (to avoid such problems in future)

**Pull request closed without merging**